rules_test.go

```diff
@@ -69,7 +69,97 @@ var rulestest = []struct {
 			path:   "/v2/domains/example.com/records/3352896//?someonetryingtobeclever%$#^&",
 			method: "WHATEVER",
 		},
-		AllowSingleDomainAllRecrodsAllActions{domain: "example.com"},
+		AllowSingleDomainAllRecordsAllActions{domain: "example.com"},
+		false,
+		true,
+	},
+	{
+		AuthorizationRequest{
+			path:   "/v2/domains/not.allowed.example.com//?/v2/domains/allowed.example.com/",
+			method: "WHATEVER",
+		},
+		AllowSingleDomainAllRecordsAllActions{domain: "example.com"},
+		false,
+		true,
+	},
+	{
+		AuthorizationRequest{
+			path:   "/v2/load_balancers/054c04f8-0b7a-40e6-a6ce-bf15b798b2ea/forwarding_rule",
+			method: "WHATEVER",
+		},
+		AllowLoadBalancersForwardingRule{lb_id: "054c04f8-0b7a-40e6-a6ce-bf15b798b2ea"},
+		true,
+		true,
+	},
+		{
+		AuthorizationRequest{
+			path:   "/v2/load_balancers/054c04f8-0b7a-40e6-a6ce-bf15b798b2ea/forwarding_rule/",
```

Comment:
I don't think this one is a valid example -> we should allow trailing slashes. Could you please make this one passing and add another testcase [non_applicable] with content at the end?